h/static/styles/partials-v2/_search.scss

```diff
@@ -62,6 +61,12 @@
   word-wrap: break-word;
 }
 
+.search-result-bucket__title-and-annotations-count {
+  display: flex;
+  flex-grow: 1;
+  width: 100%;
```

Comment:
What is the `width` property for here? The width is controlled by flexbox layout of the parent element.